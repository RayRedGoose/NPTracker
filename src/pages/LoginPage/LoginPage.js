import './LoginPage.scss'
import React from 'react'
import logo from 'assets/logo.svg'
import LoginBlock from 'containers/LoginBlock/LoginBlock'

const LoginPage = () => {
  return (
    <main className="login-page">
      <img className="logo" src={logo} alt="logo"/>
      <LoginBlock />
    </main>
  )
}

export default LoginPage
