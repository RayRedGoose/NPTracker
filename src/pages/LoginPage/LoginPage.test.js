import React from 'react';
import { shallow } from 'enzyme';
import LoginPage from './LoginPage';

it("should match snapshot with all data passed in correctly", () => {
  const page = shallow(<LoginPage />)
  expect(page).toMatchSnapshot();
});
