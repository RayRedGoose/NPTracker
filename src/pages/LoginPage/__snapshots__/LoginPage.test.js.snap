// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot with all data passed in correctly 1`] = `
<main
  className="login-page"
>
  <img
    alt="logo"
    className="logo"
    src="logo.svg"
  />
  <Connect(LoginBlock) />
</main>
`;
